sdk/cosmos/azure-spring-data-2-2-cosmos/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 ### New features
 * Updated group id to `com.azure`.
 * Updated artifact id to `azure-spring-data-2-2-cosmos`.
-* Updated azure-cosmos SDK dependency to `4.3.2-beta.1`.
+* Updated azure-cosmos SDK dependency to `4.3.2-beta.2`.
```

Comment:
These updated changes logs will only ship if you re-release the spring modules as well.